pyron-plugin-impl/src/test/resources/plugins/transformer/rules.json

```diff
@@ -249,6 +249,27 @@
             }
           ]
         },
+        {
+          "method": "GET",
+          "pathPattern": "/dyn-header-can-use-pattern-parameters-with-defined-size",
+          "requestPlugins": [
+            {
+              "name": "transform-request",
+              "conf": {
+                "body": {},
+                "headers": {
+                  "set": {
+                    "X-Account": {
+                      "path": "$body.scp",
+                      "pattern": "account-{countryCode_0_3}{a_4}-{b_4}-{c_4}-{d_4}",
```

Comment:
Is it nginx syntax with underscore as separator? I'm thinking what if we wanted to extend partern matching. I'm thinking about using regex here with simplified group naming. E.g. instead of '{countryCode_0_3}' we would have '{countryCode:REGEX-GOES-HERE}', i.e. '{countryCode:.{0,3}}'

Reply:
This way we could treat the 'pattern' as regex with simplified group naming.

Reply:
interesting :) I will think about it

Reply:
It's this syntax, because alpha-num chars plus _ are accepted in general for the param name, but later when we compute regex, group names in regex do not accept _ so, it was convenient to "abuse" it this way.